packages/cursorless-engine/src/processTargets/targets/PositionTarget.ts

```diff
@@ -44,8 +44,15 @@ export default class PositionTarget extends BaseTarget {
     if (
       this.insertionDelimiter === "\n" &&
       this.position === "after" &&
-      this.thatTarget.contentRange.isSingleLine
+      this.state.thatTarget!.contentRange.isSingleLine
```

Comment:
Are you really sure that we always have a that target?

Reply:
yeah because it's a position target, and we already use a `!` on the `thatTarget` in the constructor.  Moreover, this stuff will all get reworked in #803